apis/v1beta1/config.go

```diff
@@ -90,6 +93,18 @@ type Config struct {
 	Service    Service    `json:"service" yaml:"service"`
 }
 
+func (c *Config) GetReceivers() []parser.ComponentPortParser {
+	var receiverList []parser.ComponentPortParser
+	for name, config := range c.Receivers.Object {
+		parser, err := receiver.For(name, config)
+		if err != nil {
+			continue
+		}
+		receiverList = append(receiverList, parser)
+	}
+	return receiverList
+}
```

Comment:
I was thinking we could even introduce a full struct for "keys" or something... but that feels like maybe a slippery slope!